src/main/java/com/example/bookstore/dto/UserLoginResponseDto.java

```diff
@@ -0,0 +1,12 @@
+package com.example.bookstore.dto;
+
+import lombok.Data;
+
+@Data
```

Comment:
I'd also recommend using only specific Lombok annotations for your needs, for example Getter, Setter etc